src/mlpack/methods/ann/loss_functions/log_cosh_loss_impl.hpp

```diff
@@ -33,7 +34,13 @@ LogCoshLoss<InputDataType, OutputDataType>::Forward(
     const PredictionType& prediction,
     const TargetType& target)
 {
-  return arma::accu(arma::log(arma::cosh(a * (target - prediction)))) / a;
+  typename PredictionType::elem_type lossSum = 
+    arma::accu(arma::log(arma::cosh(a * (target - prediction)))) / a;
```

Comment:
```suggestion
  typename PredictionType::elem_type lossSum = 
      arma::accu(arma::log(arma::cosh(a * (target - prediction)))) / a;
```

Needs to be tabbed twice see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping for more details.